abci/client/client.go

```diff
@@ -81,7 +81,7 @@ type ReqRes struct {
 	*sync.WaitGroup
 	*types.Response // Not set atomically, so be sure to use WaitGroup.
 
-	mtx  tmsync.RWMutex
```

Comment:
Please confirm that this is a deliberate revert of the changes on Aug 30.

Reply:
It is, but do note we're not using this code path in the Agoric SDK.